### DISCUSSION TITLE
Typo in HTF/HTTP

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#124] [General]
Noting in the line printer...  In the error listing (I haven't configured it yet)..

Under the HTF entry at boot:

"CONFUG FILE NOT FOUND"

**Comment by KC5VDJ** [comment]
Per William's directory structure...:line#

Cml3/cmlinst:20978
Opl871/oplc/comptma:154(his):160(mine, card comptma 155)

That's "confuguration", and "confugured".

Looks to be in all of the ncctcp startup stuff on the line printer oob.

**Reply by wschaub** [reply]
It's worthwhile to fix the ncctcp typo since it's user visible. But I'm sure there's a ton of typos in the source code if you care to look for them.  from a maintenance standpoint there's probably not much to gain from fixing typos in the comments. From what I've seen from real CDC modsets it looks like a lot of typo fixes got rolled in as part of some other patch that touched the deck for functional changes.

**Reply by KC5VDJ** [reply]
Yeah, that's what I was kinda thinking.  Catalog them for later updates, they are user visible OOB.  They should be fixed, maybe in the next flurry of updates...

**Comment by kej715** [comment]
It can be argued that typos in comments of the original CDC sources have
historical value and, therefore, should not be corrected. As annoying as it
might be to read comments with spelling and typographical errors, they
almost never affect the functionality of the code (although they might
misdirect its understanding). Some CDC programmers tended to make the same
spelling and grammatical errors, and correcting those might tend to obscure
who the author was.

**Comment by kej715** [comment]
P.S.: The "CONFUG" typos in the NCCTCP sources are my errors. I've been
aware of them for years but too lazy to fix them until now. I'll fix them
in the next round of updates.
